packages/local_auth/local_auth_ios/example/ios/RunnerTests/FLTLocalAuthPluginTests.m

```diff
@@ -27,451 +27,543 @@ @interface FLTLocalAuthPluginTests : XCTestCase
 @implementation FLTLocalAuthPluginTests
 
 - (void)setUp {
-  self.continueAfterFailure = NO;
+    self.continueAfterFailure = NO;
```

Comment:
You'll probably want to set your Xcode settings (or whatever editor you are using) to 2-space indent for ObjC :)